src/dls_bluesky_core/plans/__init__.py

```diff
@@ -0,0 +1,21 @@
+from .scanspec import scan
+from .wrapped import count
+
+"""
+This package is intended to hold MsgGenerator functions which as self-contained blocks:
+start runs, collect data, and close the runs. While they may be used as building blocks
+for larger nested plans, they are primarily intended to be run as-is, and any common
+functionality which may be useful to multiple plans extracted to stubs/.
+
+Plans must adhere to BlueAPI's requirements:
+- All arguments type hinted
```

Comment:
I think this should be true of every single plan in this repository, and every other plan repository we make, and there should be some clever test that ensures this. I don't think we should be explicitly saying "type annotations matter on these plans but not those plans".

Reply:
Shouldn't mypy, and manually reviewing any type: ignore enforce the type hinting? We're enforcing type hinting on every function in the repo, so I don't know if we need an explicit test for it.

Reply:
mypy disallow_untyped_defs = true for src/ now checks that all functions have full type hinting (without making writing tests too labourious).